contrib/net/http/roundtripper.go

```diff
@@ -94,8 +95,8 @@ func (rt *roundTripper) RoundTrip(req *http.Request) (res *http.Response, err er
 		}
 	} else {
 		span.SetTag(ext.HTTPCode, strconv.Itoa(res.StatusCode))
-		// treat 5XX as errors
-		if res.StatusCode/100 == 5 {
+		// Client spans consider 4xx as errors
+		if res.StatusCode/100 == 4 {
```

Comment:
Isn't this a breaking behavioural change? Should we introduce an environment variable to allow keeping the old behaviour or would it be enough to set `DD_TRACE_HTTP_SERVER_ERROR_STATUSES` to the right values?

Reply:
@darccio , yes it is a breaking change, although in my opinion it's a "fix" as the http client spans should always have had 4xx as errors, not 5xx. 
We can discuss further.